### DISCUSSION TITLE
confirmation: Works with MFZ-KT50VG

### DISCUSSION THREAD

**Discussion by mesistostefre** [gysmo38/mitsubishi2MQTT#271] [General]
Hi,
just replaced the broken Wifi adapter by the Wemos D1 mini and this software, setup MQTT for home assistant and ... it works!

Attached to CN105 within MFZ-KT50VG .

Thanks a lot!

**Comment by fisch55** [comment]
@mesistostefre 
Hi, 
Got a Qusetion... Using ESP8266-ESP01 ---looks like the D1 mini... Do I have to set manually any Pins or not. Got the same ESP like on the pictures on the site.
THX for reply

<img width="334" alt="Bildschirmfoto 2024-08-11 um 08 55 55" src="https://github.com/user-attachments/assets/298021d1-96bd-444d-9bbc-a194a10f7fee">

**Reply by mesistostefre** [reply]
Looks good to me. Pin 0 means: use UART as described on UI.

If this does not work, maybe worth to double check if you really connected TX on ESP with RX on CN105 and RX on ESP with TX CN105.
In short:
 - RX <=> TX
 - TX <=> RX

;-)

**Reply by fisch55** [reply]
Thx a lot…. Oh ok .  
but i have 
tx- tx
rx - rx.
Thats wrong - 🙈🙈🙈🙈

**Reply by mesistostefre** [reply]
Like in real life:
RX - RX: both are listening, but nobody talks => no communication
TX - TX: both are talking, but nobody listens => no communication

RX - TX: one talks, one listens : all good

**Reply by fisch55** [reply]
Thx - Like real life Communication 🤣🤣🤣🤣. BIG THX for your HELP!!!

**Comment by fisch55** [comment]
Hello one more question .... Today came the new interior. MSZ-LN50VG2.

It has the WifiAdapter for MELCloud. Can nothing break if I connect the part to the mainboard?

I'm really a little scared 🙈.

And can I leave the MELCloud adapter connected?
